ModPatches/More Armory! Midworld Arms Pack/Patches/More Armory! Midworld Arms Pack/ThingDefs_Apparel.xml

```diff
@@ -0,0 +1,330 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+
+	<!-- === Combat Helmet === -->
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="Apparel_MA_CombatHelmet"]/statBases</xpath>
+		<value>
+			<Bulk>7</Bulk>
+			<WornBulk>3</WornBulk>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Apparel_MA_CombatHelmet"]/statBases/ArmorRating_Sharp</xpath>
```

Comment:
The helmet is stuffable and lacks a `ArmorRating_Sharp` node, so this patch fails.